components/pdfviewer/README.md

```diff
@@ -19,16 +19,16 @@ npm install @syncfusion/ej2-pdfviewer
 ## Resources
 
 * [Getting Started](https://ej2.syncfusion.com/angular/documentation/pdfviewer/getting-started/?utm_source=npm&utm_medium=listing&utm_campaign=angular-pdf-viewer-npm)
-* [View Online Demos](https://ej2.syncfusion.com/angular/demos/?utm_source=npm&utm_medium=listing&utm_campaign=angular-pdf-viewer-npm#/material/pdfviewer/default)
-* [Product Page](https://www.syncfusion.com/angular-ui-components/angular-pdf-viewer?utm_source=npm&utm_medium=listing&utm_campaign=angular-pdf-viewer-npm)
+* [View Online Demos](https://ej2.syncfusion.com/angular/demos/?utm_source=npm&utm_medium=listing&utm_campaign=angular-pdf-viewer-npm)
```

Comment:
https://ej2.syncfusion.com/angular/demos/?utm_source=npm&utm_medium=listing&utm_campaign=angular-pdf-viewer-npm#/material/pdfviewer/default - this one is correct link.  why you removed?